calculators/total_contributions_calculator.rb

```diff
@@ -17,8 +17,8 @@ def fetch
       WHERE "Filer_ID" = "FPPC"::varchar
       AND "Form_Type" = 'F460'
       AND "Line_Item" = '5'
-      AND ("Start_Date" IS NULL OR "Rpt_Date" >= "Start_Date")
-      AND ("End_Date" IS NULL OR "Rpt_Date" <= "End_Date")
+      AND ("Start_Date" IS NULL OR "From_Date" >= "Start_Date")
+      AND ("End_Date" IS NULL OR "Thru_Date" <= "End_Date")
```

Comment:
I guess this is just a bug fix.  While it should not happen, it seems like there could be a problem where the reporting period spans the Start or End date.  This would imply that the date recorded as the start/end of the campaign was not accurate, I think.  I recall these dates are set fairly arbitrarily.